pandas_gbq/gbq.py

```diff
@@ -551,23 +551,34 @@ def run_query(self, query, **kwargs):
 
     def load_data(
             self, dataframe, dataset_id, table_id, chunksize=None,
-            schema=None):
+            schema=None, progress_bar=True):
         from pandas_gbq import load
 
         total_rows = len(dataframe)
         logger.info("\n\n")
 
         try:
-            for remaining_rows in load.load_chunks(
-                    self.client, dataframe, dataset_id, table_id,
-                    chunksize=chunksize, schema=schema):
+            chunks = load.load_chunks(self.client, dataframe, dataset_id,
+                                      table_id, chunksize=chunksize,
+                                      schema=schema)
+            if progress_bar and self._check_if_tqdm_exists():
+                from tqdm import tqdm
+                chunks = tqdm(chunks)
+            for remaining_rows in chunks:
                 logger.info("\rLoad is {0}% Complete".format(
                     ((total_rows - remaining_rows) * 100) / total_rows))
         except self.http_error as ex:
             self.process_http_error(ex)
 
         logger.info("\n")
 
+    def _check_if_tqdm_exists(self):
```

Comment:
I don't mean to slow you down with tweaks. That said, there's no need for this to be a method - you can have this inline like https://github.com/pandas-dev/pandas/blob/73085773998e12e85f1044771069f63f4e8d65ad/pandas/tests/generic/test_frame.py#L23